### ISSUE TITLE
Create Knative CRD Icons for use in diagrams and UI

### ISSUE THREAD

**Issue by dgerd** [knative/community#44]
It would be great to have official icons for each Knative CRD (Service, Revision, Channels, Brokers, etc.) so that they can be used in service architecture diagrams and UIs to represent Knative resources.

Creating this issue in community as:
1. Icons seem like "community material"
2. Likely needs Steering Committee support as it relates to the Knative brand

cc @bbrowning @mattmoor

**Comment by evankanderson** [comment]
Probably also needs to find a graphic designer if we want them to look good.

**Comment by dprotaso** [comment]
This is probably a trademark issue now

**Comment by abrennan89** [comment]
Docs probably have dependency on this since we would like to have good diagrams for our upcoming About Knative page and other conceptual pages in the website. Any plans to progress this?

**Comment by zroubalik** [comment]
+1 to have this, I guess CNCF can help with this.

**Comment by aliok** [comment]
Is this still relevant?

We might ask the UX WG to guide us with this, or to reach out to CNCF for the community, but do we still want this?

cc @knative/ux-wg-leads cc @knative/steering-committee

**Comment by Cali0707** [comment]
UX WG can definitely add the design of icons to our backlog, if that is how SC and others want to proceed with this @aliok

**Comment by zainabhusain227** [comment]
@Cali0707 @aliok , I agree with having the UX group do the icon design. I'd be interested in working on this task!

**Comment by aliok** [comment]
Ok, added this ticket to TOC backlog.
cc @knative/technical-oversight-committee

**Comment by dprotaso** [comment]
Moving to steering - as this sorta falls under branding/marketing

**Comment by nainaz** [comment]
@knative/steering-committee please add your inputs.
We can decide on the outcome based on everyone's thoughts

**Comment by dprotaso** [comment]
@evankanderson you were looking into this a bit - did you ever settle on anything?

**Comment by evankanderson** [comment]
We did not, but I agree that these would be very useful.  I'm not especially graphically inclined but I might get inspired to do a sketch or two in the next week. 

Items we would want to represent: 

## Serving

- Service
- Route
- Configuration
- Revision
- instance / Pod
- (possibly) gateway / kingress

## Eventing

- Broker
- Trigger
- Source
- Channel
- Subscription 
- (possibly) Sink
- (possibly) Event Registry

## Functions

- Function
- Language Pack

Stuff that may not need its own icon, or needing different treatment (e.g. action icon on arrows, rather than a "logo"):

- Addressable
- event delivery
- cold start / activator-handled request

**Comment by dprotaso** [comment]
Note k8s icons - https://github.com/kubernetes/community/tree/master/icons
